includes/definitions/discovery/adva_xg300.yaml

```
@@ -0,0 +1,96 @@
+mib: OA-SFP-MIB:DEV-CFG-MIB
+modules:
+    os:
+        sysDescr_regex: '/(?<hardware>FSP150-XG3\d+\s\w+\s\w+-\w) Ver\. (?<version>[\d+_]+\d \(\d+\)$)/'
```

Maybe nitpicking but I'd suggest using something a bit more ... forgiving:
```suggestion
        sysDescr_regex: '/(?<hardware>.+) Ver\. (?<version>[^)]+\))/'
```

---

Ok, I could've clicked a button and fixed that. Oh well, I put it in.